components/wpa_supplicant/src/tls/rsa.c

```diff
@@ -239,93 +533,178 @@ size_t crypto_rsa_get_modulus_len(struct crypto_rsa_key *key)
  * Returns: 0 on success, -1 on failure
  */
 int crypto_rsa_exptmod(const u8 *in, size_t inlen, u8 *out, size_t *outlen,
-		       struct crypto_rsa_key *key, int use_private)
+               struct crypto_rsa_key *key, int use_private)
```

Comment:
this section is a good example of how whitespace *changes* can create a mess. You really only want to insert a block of code, but now it shows as a complex diff, because, you added some new code *and* changed the whitespace in a bunch more code. 

so, in summary
1. please make sure your new code/additions has clean whitespaces
2. please don't change existing whitespace (even if it is broken) in lines that you don't touch